app/components/ui/tale-tabs-selector/template.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="ui">
+  <div class="ui secondary pointing menu" style="display:inline-flex;">
+      <a class="item {{if activeTabInteract "active"}}" data-tab="tab-interact" onclick={{action "activateInteract"}}>Interact</a>
+      <!--<a class="item {{if activeTabFiles "active"}}" data-tab="tab-files" onclick={{action "activateFiles"}}>Files</a>-->
+      <a class="item {{if activeTabMetadata "active"}}" data-tab="tab-metadata" onclick={{action "activateMetadata"}}>Metadata</a>
+  </div>
+  <div class="ui tab {{if activeTabInteract "active"}}" data-tab="tab-interact">
+      {{#if activeTabInteract}}
+        {{ui/tale-tab-interact model=model}}
+      {{/if}}
+  </div>
+  <div class="ui tab {{if activeTabFiles "active"}}" data-tab="tab-files">
+      {{#if activeTabFiles}}
+        {{ui/tale-tab-files model=model}}
+      {{/if}}
+  </div>
+  <div class="ui tab {{if activeTabMetadata "active"}}" data-tab="tab-metadata" style="overflow-x:auto;max-height:50vh;">
```

Comment:
<img width="1301" alt="screen shot 2018-12-06 at 8 37 36 am" src="https://user-images.githubusercontent.com/20361247/49594455-ea126800-f943-11e8-9de3-dabb565d7b38.png">

Reply:
Any tips for how to get the area with the scroll bar to resize correctly with screen size? `height: 100 %` doesn't have the desired effect, as the scroll bar doesn't show up if `max-height` isn't set.

Reply:
Also, @bodom0015, for the height I have defined Sass variables so we can keep track of the different elements that make up the app in this case:
```
        $inverted-menu-height: 48px;
        $main-menu-height: 0; // was 64px
        $wt-paddleboard-panel-height: 50px;
        $wt-main-row-height: 52px;
        $bottom-separation: 2.5rem;
        $footer-height: 94px;
        $left-panel-padding: 38px;
        $wt-paddleboard-panel-tall-height: 63px;
```
...and then I assign that panel's height by doing `height: calc(100vh - (<sum_of_other_ elements_heights_variables>))`;